templates/web/pages/user_form/user_form_page.tt.html

```diff
@@ -45,7 +45,9 @@
 
     <!-- Start form -->
 
-    <form method="post" action="/cgi/user.pl" enctype="multipart/form-data" id="user_form">
+<div class="donation-banner__content">
```

Comment:
Why do we have a class for the donation-banner ?

Reply:
Because the css is same i.e. for smaller screens we want the flex-direction to be column instead of row, so instead of writing separate css for this div container, i reused the css for donation-banner class..
If you want separate css for this div that we can even do it in that way too!

Reply:
Maybe it's better to duplicate the class @MonalikaPatnaik so that it's more meaning full (you can add the class to same css rules as donation-banner__content.

Reply:
Done @alexgarel ✅